bigbluebutton-html5/imports/ui/components/poll/live-result/component.jsx

```diff
@@ -201,12 +208,14 @@ class LiveResult extends PureComponent {
           </div>
           {pollStats}
         </div>
-        {currentPoll && currentPoll.answers.length > 0
+        {currentPoll && currentPoll.answers.length >= 0
```

Comment:
I think this extra symbol is likely the key change of the PR

Reply:
Exactly! For the User Response Poll, there is no predefined answers, so `currentPoll.answers.length` is equal to 0 in the beginning.